TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -545,7 +545,7 @@ public static void CallQueuedCommand(string user, bool isWhisper, [Group(1)] boo
 	/// <name>Call All</name>
 	/// <syntax>call all</syntax>
```

Comment:
```suggestion
	/// <syntax>callall</syntax>
```
Could you update the syntax as well so the documentation doesn't include a space between call and the command?